src/common/utils/getFundOptions.js

```diff
@@ -1,5 +1,8 @@
+import { get } from 'lodash';
+import { DICT_FUNDS } from '@folio/stripes-acq-components';
+
 // eslint-disable-next-line import/prefer-default-export
-export const getFundOptions = (funds = []) => funds.map(fund => ({
+export const getFundOptions = (resources) => get(resources, [DICT_FUNDS, 'records'], []).map(fund => ({
```

Comment:
what is the idea behind this change?

Reply:
Firstly, to avoid an error:
![image](https://user-images.githubusercontent.com/38653807/80094424-e26e5600-856e-11ea-9de2-8080243cab55.png);
Secondly, a similar approach was used for the container component which was provided as an example.
![image](https://user-images.githubusercontent.com/38653807/80094886-b56e7300-856f-11ea-8ea8-9fa34d82681c.png)

Reply:
first of all, you shouldn't copy-paste just to make something similar - you should realize what you're doing. so the second point is not valid, please provide why you want to make this here.
regarding error - IMO, when you come to the code base you're not familiar with the best way to not break things is to make as less changes as possible. That's why I don't see any point to change the View component, it was working before. What you can do - is to avoid such error in Container. Obviously it's occurred when you're trying to render presentation component while list of options still loading from the back-end. Maybe it makes sense to not render presentation component while it's happening.

Reply:
Thanks for the advice, it helped me to resolve the issue of data loading into Container.
Regarding the question, I made it there because I was trying to fix the error and found out this approach in the example. As a result, it helped me to avoid this error.